checks/linuxHardening.yaml

```diff
@@ -0,0 +1,89 @@
+successMessage: One of AppArmor, Seccomp, SELinux, or dropping Linux Capabilities are used to restrict containers using unwanted privileges
+FailureMessage: Use one of AppArmor, Seccomp, SELinux, or dropping Linux Capabilities to restrict containers using unwanted privileges
+category: Security
+target: Container
```

Comment:
Here the target is `Container` but you are checking `SecompProfile` from PodSpec.SecurityContext and not checking for `Containers.SecurityContext.` was this on purpose?

Reply:
The per-container schema is requiring `securityContext` depending on the pod-level field. I commented with more detail in line 30 of this file.